docs/resources/custom_policy_template.md

```diff
@@ -0,0 +1,152 @@
+---
+Title: "Custom Policy Template Resource"
+Description: |-
+   Creating a custom policy template.
+---
+
+# Custom Policy Template Resource
+
+This resource enables users to create custom custom policy template in TMC.
+
+For more information regarding custom policy template, see [Custom Policy Template][custom-policy-template].
+
+[custom-policy-template]: https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-F147492B-04FD-4CFD-8D1F-66E36D40D49C.html
+
+## Example Usage
+
+```terraform
+resource "tanzu-mission-control_custom_policy_template" "sample" {
```

Comment:
No documentation file added for data source?

Reply:
Data source is not yet created because it needs code from the custom policy template assignment.
The data source will be created once this is merged to main and can be used to rebase the custom policy assignment branch.